app/questions/page.js

```diff
@@ -1,7 +1,8 @@
-import styles from '../styles/SuggestionsPage.module.css';
-import LineBreak from '../components/LineBreak';
-import LogoBanner from '../components/LogoBanner';
-import ContactForm, { formTypes } from '../components/ContactForm';
+import styles from '../../styles/SuggestionsPage.module.css';
+import LineBreak from '../../components/LineBreak';
+import LogoBanner from '../../components/LogoBanner';
+import ContactForm from '../../components/ContactForm';
+import { formTypes } from '../../components/constants';
```

Comment:
Had to move this into a `constants` file, I was getting an error when trying to import from a client component:

```
"you cannot dot into a client module from a server component"
```